cypress-io/cypress
# feat: add `CYPRESS_SKIP_VERIFY` option to suppress verification

- Closes #22243

### Additional details
The PR adds a flag `CYPRESS_SKIP_VERIFY` to prevent the verification steps from running. These steps are semi-redundant in CI environments, and regularly timeout leading to false negatives in test suites. 

### Steps to test
- Delete the Cypress cache on your machine and run Cypress with `CYPRESS_SKIP_VERIFY=true` in your env
- Confirm that the verification messages are not shown ("Looks like this is your first time running Cypress N.N.N...")

### How has the user experience changed?
If a user opts in they will not get the verification on Cypress startup

### PR Tasks
